src/external/bson/CMakeLists.txt

```
@@ -0,0 +1,52 @@
+set(BSON_SRC
```

```suggestion
include(CheckFunctionExists)
include(CheckStructHasMember)

set(BSON_SRC
```

I think it's better if we include CMake modules where we need them - it's not harmful if several different listfiles in the same project include the same modules.